bundles/framework/layerlist/view/LayerViewTabs/SelectedLayers/LayerBox/Footer/OpacitySlider.jsx

```diff
@@ -14,6 +14,19 @@ const Border = styled('div')`
 `;
 const StyledSlider = styled(Slider)`
     margin: 0 !important;
+    .ant-slider-track {
+        background-color: #0091ff;
+    }
+    .ant-slider-handle {
+        border: #0091ff solid 2px;
+        margin-top: -6px;
+    }
+    &:hover .ant-slider-track {
+        background-color: #003fc3 !important;
+    }
+    &:hover .ant-slider-handle {
+        border: #003fc3 solid 2px !important;
+    }
```

Comment:
Could we use src/react/components/Opacity.jsx here instead so we don't have to assign style twice (here and in src/react/components/Opacity.jsx)? It's currently used in bundles\admin\admin-layereditor\view\AdminLayerForm\VisualizationTabPane\Opacity.jsx but since it's a common component we should be able to use it here as well.